math-libs/BLAS/CMakeLists.txt

```diff
@@ -78,6 +75,7 @@ if(_enable_rocRoller)
       therock-yaml-cpp
       # therock-cl11 # will need this dep to enable client
     RUNTIME_DEPS
+      therock-catch2
```

Comment:
It should be either a build dep or a runtime dep but not both (if it is a runtime dep, it will be available at build time). As Stella's commented, if building as a static lib, it will be a build dep (as we do not need to ship any shared objects).

```suggestion
```